### PR TITLE
Enhance a look of the example in Getting Started

### PR DESCRIPTION
**Pull request by soupault**
This is how it looks to me at the moment:
image

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -41,7 +41,8 @@ visit our `gallery </docs/dev/auto_examples>`__.
 
         from skimage import data, io, filters
 
-        image = data.coins() # or any NumPy array!
+        # ... or any other NumPy array!
+        image = data.coins()
```
Comment (jni) [inline index.rst]: I'd reverse this and put the comment underneath — it reads much more like English.

Comment (soupault) [inline index.rst]: I have tried that also, but it doesn't seem logical (comment usually address the line beneath).
Let's wait for more opinions.

Comment (jni) [inline index.rst]: @soupault I agree that we should hear from others, but my suggested fix would be to put a newline after the comment.

Comment (stefanv) [inline index.rst]: The comment should be after the assignment to image, right?

**Comment by soupault** [comment]
Alright, I don't mind.
Is below now.

**Comment by jni** [comment]
👍  from me

**Pull request merged**